kong/plugins/session/strategies/postgres/session_metadatas.lua

```diff
@@ -0,0 +1,14 @@
+local fmt = string.format
```

Comment:
Add the license header in this new file. Same for other new files.

```
-- This software is copyright Kong Inc. and its licensors.
-- Use of the software is subject to the agreement between your organization
-- and Kong Inc. If there is no such agreement, use is governed by and
-- subject to the terms of the Kong Master Software License Agreement found
-- at https://konghq.com/enterprisesoftwarelicense/.
-- [ END OF LICENSE 0867164ffc95e54f04670b5169c09574bdbd9bba ]
```

Reply:
I don't think so.
This license header is only for the enterprise. Right?

Reply:
You are right. Only needed when cherry pick.

Reply:
CE repo doesn't need this header.